modules/ROOT/pages/administration/databases.adoc

```diff
@@ -903,6 +925,76 @@ ALTER DATABASE nonExisting IF EXISTS SET TOPOLOGY 1 PRIMARY 0 SECONDARY
 [role="statsonlyqueryresult"]
 0 rows
 
+[role=enterprise-edition]
+[[administration-databases-alter-database-options]]
+=== Options
+
+The `ALTER DATABASE` command can be used to set or remove specific options.
```

Comment:
should definitely mention that this is a subset of the options available for CREATE

Reply:
Isn't the table below enough?

Reply:
While the table shows it is feels nice to point it out explicitly in the text as well (otherwise I'd probably still try setting some of the create thinking they've just been missed)